agency-api-spec.md

```diff
@@ -25,6 +26,37 @@ This draft does not address:
 * size or rate limits
 * error message/status code related to exceeded the rate limit
 * any subsequent calls to the internal FOIA.gov API (to capture info needed to subsequently retrieve status, for example)
+* detailed security controls
+
+
+### Security controls
+
+Once we have confidence in our data models and the touch points for
+interoperability, we will be working with the security team at DOJ to ensure the
+Portal meets the federal requirements for security controls. For now, we include
+only some preliminary ideas based on the agency feedback we’ve received.
+
+Each agency will be responsible for implementing security controls on their own
+end as per their agency regulations and authority to operate. This may include
+configuration of a web application firewall, anti-virus scanning of
+file attachments, and validation of HTTP headers.
+
+
+#### HTTPS
+
+Agency endpoints should be restricted to HTTPS only with valid TLS certificates.
+The Portal will validate your certificate as part of the HTTPS request.
```

Comment:
@adborden are we saying we won't POST to an endpoint if it's HTTP?

Reply:
@malikkotob right, we cannot transmit the request in the clear since it contains PII. We also should check the certificate is valid as part of that (which should be the default in most HTTP libraries anyway).

Reply:
@adborden we don't have code in place to enforce that it's an HTTPS endpoint. Maybe this is something that can be handled in communication when onboarding agencies, in the event that the spec isn't followed and an HTTP endpoint is provided to whoever is handling onboarding. (The library does check for valid certs).